### PR TITLE
sysdeps/linux: Implement sys_clock_getres

### PR DESCRIPTION
**Pull request by Dennisbonke**
Part of mlibc LFS project.

### CODE REVIEW

File: sysdeps/linux/include/bits/syscall.h [hunk 1]
```diff
@@ -9,6 +9,7 @@ typedef long __sc_word_t;
 
 // These syscall macros were copied from musl.
 #define __scc(x) ((__sc_word_t)(x))
+#define __syscall0(n) __do_syscall0(n)
```
Comment (avdgrinten) [inline sysdeps/linux/include/bits/syscall.h:12]: This change seems to be unrelated.

File: sysdeps/linux/x86_64/abi-bits/arch-syscall.h [hunk 2]
```diff
@@ -58,8 +58,8 @@
 #define SYS_setpriority 141
 #define SYS_arch_prctl 158
 #define SYS_setrlimit 160
-#define SYS_sys_futex 202
```
Comment (avdgrinten) [inline sysdeps/linux/x86_64/abi-bits/arch-syscall.h:61]: Why is this done?

**Review by Geertiebear — APPROVED** [review]
